pkg/maglev/maglev.go

```diff
@@ -64,8 +75,6 @@ func getOffsetAndSkip(backend string, m uint64) (uint64, uint64) {
 
 func getPermutation(backends []string, m uint64, numCPU int) []uint64 {
 	var wg sync.WaitGroup
-	bCount := len(backends)
```

Comment:
If `len(backends) > maxBackends`, then I'd just allocate a slice for this request.

Reply:
Done